### ISSUE TITLE
How edit exisiting sketch pattern

### ISSUE THREAD

**Issue by imahdio** [imahdio/SOLIDWORKS-2022-Essential-Training#9]
I'm studying  Creating repeating patterns in a sketch lesson from [here](http://20.113.148.110/solidworks-2022-essential-training/06-Modifying%20Sketches/)

here is the direction of sketch pattern on a sketch

instruction|image
-|-
1-select whole shape|![image](https://user-images.githubusercontent.com/64577273/163806679-6b25e5f0-a40e-4361-88f8-5da3ae0813cb.png)
2-select sketch pattern from toolbar|![image](https://user-images.githubusercontent.com/64577273/163807984-2c27ae51-489f-40e1-a218-401f081182d1.png)
3-sumbit the customizations by click OK|![image](https://user-images.githubusercontent.com/64577273/163808045-c26c53b6-194b-4c2a-8f51-1389fdaea6a6.png)

## the question is:
in above assumption, what should I do if I want to add 4th parallelugram shape with new angel between from 40 degree to 50 ?

**Comment by imahdio** [comment]
https://youtu.be/hoDEIO6Pl8o?t=2354
there are 2 assistive hints beside the origin whcih help us to modify the angel and number of instances

**Comment by Tundrees** [comment]
Yes when the patterned entity is double-clicked, you will find the number of pattern on the feature manager.
